ansible/psi/deploy-all.sh

```diff
@@ -5,9 +5,12 @@ set -eu
 # cd to ansible/
 cd "$(dirname $(dirname $(realpath $0)))"
 
+# first instance name; can be set by env
+FIRST=${FIRST:-1}
+
 # number of instances; limited by quota
 NUM=35
 
-seq $NUM | parallel --line-buffer -j4 ansible-playbook -i inventory/ -e instance_name='rhos-01-{}' psi/launch-tasks.yml
+parallel -i -j4 ansible-playbook -i inventory/ -e instance_name='rhos-01-{}' psi/launch-tasks.yml -- $(seq $FIRST $NUM)
 
-ansible-playbook -i inventory/ psi/image-cache.yml
+ansible-playbook -f20 -i inventory/ psi/image-cache.yml
```

Comment:
non-blocking nit, `--forks` would be what I prefer reading here. 20 sounds like a lot then again never seen ansible-playbook runs "eat resources" with the default 5.

Reply:
On the client side it's really just a bunch of `ssh` processes. I find 20 to be just fine for all our cockpituous cases, we don't do anything expensive on the caller side.

But I'll change to `--forks` indeed.